client/components/track-interactions/README.md

```diff
@@ -48,9 +48,9 @@ clicking yields:
 **`mapPropsToAction`** (function): for cases where `fields` isn't flexible enough:
 
 ```js
-const mapPropsToAction = props => ( {
-	name: props.theme.id,
-	isCurrentTheme: props.theme.id === getCurrentTheme( state ),
+const mapPropsToAction = ( state, { theme } ) => ( {
```

Comment:
@mcsf I've made `state` explicit here b/c `getCurrentTheme` was using it ^ (and now `isActiveTheme` is), but looking at https://github.com/Automattic/wp-calypso/blob/master/client/components/track-interactions/index.js#L53 it looks like we aren't even passing `state` in. So... what do we do about this example? 😄